### ISSUE TITLE
Webpack config not found when using es6 export statement

### ISSUE THREAD

**Issue by giannif** [trayio/babel-plugin-webpack-alias#12]
In `webpack.config.babel.js` 

If I use:

```javascript
export default {
 // webpack config properties
}
```

babel-plugin-webpack-alias's `getConfig` fails because it needs to reference `default` (as of [Babel 6](https://phabricator.babeljs.io/T2212))

Perhaps checking the `__esModule` could help determine if we should use `default`
```javascript
function getConfig(configPath, findConfig) {
    var conf;
    if(!findConfig) {
        // Get webpack config
        conf = require(resolve(process.cwd(), configPath));
    } else {
        conf = require(findUp.sync(configPath));
    }
    if (conf && conf.__esModule) {
        conf = conf.default
    }
    return conf;
}
```

**Comment by sebinsua** [comment]
For those reading these issues, wondering how they can use the fixes that haven't yet been merged, I have a fork of this plugin that I've been working on which works a lot better for me: [`babel-plugin-webpack-aliases`](https://github.com/sebinsua/babel-plugin-webpack-aliases)

**Comment by giannif** [comment]
@sebinsua this is awesome! Maybe you could mention the new name :) I didn't realize it right away

**Comment by adriantoine** [comment]
@sebinsua would you mind making a PR to this repo?

**Comment by adriantoine** [comment]
A fix has been made and it is included in [v1.7.0](https://github.com/trayio/babel-plugin-webpack-alias/releases/tag/v1.7.0)!
